main/res/layout/emojiselector.xml

```diff
@@ -4,30 +4,27 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <!--
-    <LinearLayout
-        android:id="@+id/emoji_groups"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal" />
-
-    -->
```

Comment:
Very cool, I really like it! Only one little thing, please remove this comment if the code is not needed anymore...

Reply:
it's a leftover from an earlier PR and removed by this PR, so it's marked red

Reply:
Oh, damn! I'm so sorry. I thought it was added in the PR... It was probably already too late last night 😅

Reply:
No problem :-)